tools/autograd/derivatives.yaml

```diff
@@ -472,6 +472,9 @@
 - name: erfc(Tensor self) -> Tensor
   self: -2.0 / sqrt(M_PI) * exp(-(self.pow(2))) * grad
 
+- name: erfcx(Tensor self) -> Tensor
+  self: (2.0 * self * result - 2.0 / sqrt(M_PI)) * grad
```

Comment:
Would you elaborate (in response to this comment) on how you derived this formula?